tests/mono-binds/test02.icry.stdout

```diff
@@ -17,6 +17,7 @@ test02::test =
 
 
 
+
```

Comment:
It's unfortunate that there is so much extra whitespace printed here (and in other test cases) between lines. This is a problem that has always existed, but it's even more noticeable now because...